content/guides/nextjs/github-open-authoring/hosting-vercel.md

```diff
@@ -15,18 +15,18 @@ Finally, we're ready to get this app in production. Make sure all the latest cha
    5. It will auto-detect that your using Next.js. The default build, output, and development commands don't need to be changed.
```

Comment:
```suggestion
   5. It will auto-detect that you're using Next.js. The default build, output, and development commands don't need to be changed.
```